tools/run_tests.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+ # run setup script
+node ./tools/loadTestFiles.js && 
```

Comment:
I'd store the executables for `node` in a variable so that if we need an absolute path on some system, we can use `$node`